blockchain/fullblocktests/generate.go

```diff
@@ -1931,6 +1943,150 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	})
 	rejected(blockchain.ErrScriptMalformed)
 
+	// Create block that spends immature change from one SStx in another SStx.
```

Comment:
Please avoid the SStx terminology and call it them ticket purchases.  We've been trying to eradicate that terminology from the code base because it's confusing.